### DISCUSSION TITLE
Extensions

### DISCUSSION THREAD

**Discussion by mrelhussein** [gismo/gismo#526] [Q&A] [answered]
How can I integrate the extensions (gsKLShell, gsElasticity ...) in my  gismo ?

Thank you in advance

**Comment by hverhelst** [comment, marked as answer]
Dear @mrelhussein , 

You can have a look at the repositories of the extensions: [github.com/gismo/gsKLShell/](github.com/gismo/gsKLShell/), [github.com/gismo/gsElasticity/](github.com/gismo/gsElasticity/) et cetera and follow the installation instructions there.
